ansible_collections/arista/avd/molecule/eos_cli_config_gen/documentation/devices/router-internet-exit.md

```diff
@@ -0,0 +1,94 @@
+# router-internet-exit
+
+## Table of Contents
+
+- [Management](#management)
+  - [Management Interfaces](#management-interfaces)
+  - [Router Internet Exit](#router-internet-exit)
+
+## Management
+
+### Management Interfaces
+
+#### Management Interfaces Summary
+
+##### IPv4
+
+| Management Interface | Description | Type | VRF | IP Address | Gateway |
+| -------------------- | ----------- | ---- | --- | ---------- | ------- |
+| Management1 | oob_management | oob | MGMT | 10.73.255.122/24 | 10.73.255.2 |
+
+##### IPv6
+
+| Management Interface | Description | Type | VRF | IPv6 Address | IPv6 Gateway |
+| -------------------- | ----------- | ---- | --- | ------------ | ------------ |
+| Management1 | oob_management | oob | MGMT | - | - |
+
+#### Management Interfaces Device Configuration
+
+```eos
+!
+interface Management1
+   description oob_management
+   vrf MGMT
+   ip address 10.73.255.122/24
+```
+
+### Router Internet Exit
+
+#### Exit Groups Summary
+
+| Exit Groups | Local Connections | Fib Default |
+| ----------- | ----------------- | ----------- |
+| eg_01 | - | - |
+| eg_02 | - | True |
+| eg_03 | eg_03_lo_01 | - |
+| eg_03 | eg_03_lo_02 | - |
+| eg_03 | - | True |
```

Comment:
This is deceitful - seems like fib default is not configured for eg_03 when it actually is.